test/configCases/source-map/extract-source-map2/a.js

```diff
@@ -0,0 +1,3 @@
+const a = 1;
+// comment
+//#sourceMappingURL=/a.map
```

Comment:
Let's add a test for remote source maps, i.e. `http://github.com/webpack/webpack/test/configCases/source-map/my-source-map.js.map`